torch/linalg/__init__.py

```diff
@@ -1245,7 +1305,7 @@
 
 For :attr:`p` is one of `('fro', 'nuc', inf, -inf, 1, -1)`, this function uses
 :func:`torch.linalg.norm` and :func:`torch.linalg.inv`.
-As such, in this case, the matrix (or every matrix in the batch) :attr:`input` should be square
+As such, in this case, the matrix (or every matrix in the batch) :attr:`A` should be square
```

Comment:
"should" is a weird word for documentation

What happens if the matrices aren't square are aren't invertible?

Also, are we consistent about use of the word "invertible"?

Reply:
Yes, we are consistent. If it's not invertible we raise an error. This is documetned in the `Raises` section.

I'm changing "should be" to "has to be".